frb_example/pure_dart/test/api/feature_test.dart

```diff
@@ -0,0 +1,13 @@
+// FRB_INTERNAL_GENERATOR: {"forbiddenDuplicatorModes": ["rustAsync", "sse", "sync sse", "rustAsync sse"], "replaceCode": {"pseudo_manual/misc_no_twin_example_a_twin_sync": "misc_no_twin_example_a"}}
+
+import 'package:frb_example_pure_dart/src/rust/frb_generated.dart';
+import 'package:test/test.dart';
+import 'package:frb_example_pure_dart/src/rust/api/misc_no_twin_example_a.dart';
+
+Future<void> main({bool skipRustLibInit = false}) async {
```

Comment:
nit: since you are using misc_no_twin_example_a.rs, just write test code in misc_no_twin_example_a_test.dart (instead of the new file feature_test.dart)